templates/registration/contact_thanks.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
     <h1>Contact</h1>
-    <div class="contact-thanks">
+        <div class="contact-thanks">
     <p>Thank you! Your message was received successfully.</p>
```

Comment:
Shoot. Misread your last comment. Thought the plus in front of it meant that was the line you wanted to indent.